test/http/host_meta.js

```diff
@@ -0,0 +1,41 @@
+var uuid      = require('uuid')
+var Promise   = require('bluebird')
+var codius    = require('../../')
+var assert    = require('assert')
+var supertest = require('supertest')
+var server    = require('../../lib/application')(codius)
+var http      = supertest(server)
+var fs        = require('fs')
+var path      = require('path')
+
+describe('Host Meta and Root Endpoints', function() {
+  var version, publicKey
+
+  before(function(done) {
+    fs.readFile(path.join(__dirname+'/../../package.json'), function(error, file) {
+      version = JSON.parse(file.toString()).version
+      publicKey = fs.readFileSync(__dirname+'/../../server.crt').toString()
```

Comment:
It looks like this path doesn't work with circleci, which does this:
`yes US | openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /tmp/contracts/server.key -out /tmp/contracts/server.crt`
then
`SSL_KEY=/tmp/contracts/server.key SSL_CERT=/tmp/contracts/server.crt CONTRACTS_STORAGE=/tmp/contracts/ npm test`